cosmo_tester/config_schemas/platform_openstack.yaml

```diff
@@ -4,7 +4,7 @@ plugin_package_name:
   default: cloudify-openstack-plugin
 plugin_version:
   description: The expected version of this plugin.
-  default: 3.2.10
+  default: 3.2.18
```

Comment:
Is .18 the latest version?

Reply:
adopting the regex approach

Reply:
Are you intending that to be in this PR or another?

Reply:
In the current one